### DISCUSSION TITLE
Implementation of comparison data in line chart fluent ui @microsoft/charting-team

### DISCUSSION THREAD

**Discussion by saniakhalid2** [microsoft/fluentui#31993] [Q&A]
1. Is it possible to customize tooltip in such a way where i can show data of two different dates for same legend ,I am trying to implement onrendercalloutdatapoint ,but i doubt if it is possible to implement comparison data,?
2. is it possible to show two dates for comparison on x axis on same point ?
3. Y axis customization , is it possible to hide y axis nd show two y axis ,currently i am aware that we can show primary and secondary y axis but is it possible to hide them both?
Please guys share your opinion on this

**Comment by saniakhalid2** [comment]
@microsoft/charting-team

**Reply by AtishayMsft** [reply]
Taken offline with Sania.
